### ISSUE TITLE
Screenreader unable to read input validation error in Shadow DOM

### ISSUE THREAD

**Issue by johnheroy** [PolymerElements/paper-input#209]
Tested with the ChromeVox extension, paper-input validation errors seems to work fine in Shady DOM (as in the demo) but unable to read that the paper-input is a paper-input and the validation error-message under Shadow DOM in Chrome.

**Comment by notwaldorf** [comment]
ChromeVox is really really broken with the Shadow DOM (it will not tell you the right thing). I've tested VoiceOver with the Shadow DOM and it's fine (though a bit verbose), so I believe there's nothing we can do here until ChromeVox is fixed

/cc @cdata

**Comment by felixzapata** [comment]
hi @notwaldorf and @cdata , any news when Chromevox extension will be fixed to work fine with Shadow DOM?
